regression/QF_LIA/prp-0-12_simplified_3.smt2

```diff
@@ -0,0 +1,6 @@
+(set-logic QF_LIA)
+(declare-fun v0 () Int)
+(declare-fun v2 () Int)
+(assert (= (ite (= v2 0) (ite (= v0 0) 1 0) (ite (= (ite (= v0 0) 1 0) 0) 1 0)) 0))
+(check-sat)
+(exit)
```

Comment:
Could you add also a similar regression test with expected answer UNSAT?

Reply:
Done.